components/post_view/post_info/index.js

```diff
@@ -4,18 +4,24 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {addReaction, removePost} from 'mattermost-redux/actions/posts';
+import {makeGetChannel} from 'mattermost-redux/selectors/entities/channels';
```

Comment:
Can we use [getChannel](https://github.com/mattermost/mattermost-redux/blob/master/src/selectors/entities/channels.js#L87) directly?

Reply:
You are right again. Thanks.

Reply:
Fixed